ipatests/prci_definitions/nightly_master.yaml

```diff
@@ -1192,3 +1192,15 @@ jobs:
         template: *ci-master-f29
         timeout: 3600
         topology: *ipaserver
+
+  fedora-29/test_ntp_options:
```

Comment:
Totally a nitpick: could you move this between `fedora-29/external_ca_TestMultipleExternalCA` and `fedora-29/test_pkinit_manage`? This will help with future comparisons with master branch.

Reply:
Sure, done.